index.js

```diff
@@ -140,7 +142,7 @@ Marigold.AttributeMap = function() {
   }
 }
 
-Marigold.MessageImpressionType = {StreamView: 1, DetailView: 2, InAppView: 0};
+MessageStream.MessageImpressionType = {StreamView: 1, DetailView: 2, InAppView: 0};
 Marigold.DeviceValues = {Attributes: 1, MessageStream: 2, Events: 4, ClearAll: 7};
```

Comment:
We can remove the `Attributes` value here as this is no longer handled in the `clearDevice` method.